src/server/req_jobobit.c

```diff
@@ -649,7 +651,14 @@ rel_resc(job *pjob)
 
 	/* Mark that scheduler should be called */
 
-	set_scheduler_flag(SCH_SCHEDULE_TERM);
+	if (find_assoc_sched_jid(pjob->ji_qs.ji_jobid, &psched))
+		set_scheduler_flag(SCH_SCHEDULE_TERM, psched);
+	else {
+		pbs_queue *pq;
+		pq = find_queuebyname( pjob->ji_qs.ji_queue);
+		sprintf(log_buffer, "Unable to reach scheduler associated with job %s", pq->qu_attr[QA_ATR_partition].at_val.at_str);
```

Comment:
Design document states that the message gets printed with a job id not partition id.

Reply:
Have updated the design document.

Reply:
well, the message still needs to change to say "partition" instead of "job"